src/mainboard/protectli/vault_jsl/mainboard.c

```diff
@@ -192,7 +192,7 @@ static int smbios_type41_write_ethernet_behind_switch(struct device *root_port,
 		return 0;
 
 	/* Loop through all downstream ports of the switch */
-	for (child = switch_dev->bus->children; child; child = child->sibling) {
+	for (child = switch_dev->upstream->children; child; child = child->sibling) {
```

Comment:
I think this (topmost commit) can be cleanly squashed into https://github.com/Dasharo/coreboot/pull/708/commits/c7aa69f948e264b6217318f29059e6f0f1cd3395.

Reply:
squashed to https://github.com/Dasharo/coreboot/pull/708/commits/d831fca158f9a74a7150a71f78469e00a6586f4f